scripts/linters/general_purpose_linter_test.py

```diff
@@ -231,16 +231,6 @@ def test_invalid_use_of_request(self):
         self.assertEqual('Bad pattern', lint_task_report.name)
         self.assertTrue(lint_task_report.failed)
 
-    def test_invalid_use_of_object(self):
-        linter = general_purpose_linter.GeneralPurposeLinter(
-            [INVALID_OBJECT_FILEPATH], FILE_CACHE)
```

Comment:
Remove variable `INVALID_OBJECT_FILEPATH` from the top of file if it is not used anywhere else and delete the file as well.

Reply:
Done.